README.md

```diff
@@ -5,14 +5,19 @@ Copyright 2018-present Open Networking Foundation
 SPDX-License-Identifier: Apache-2.0
 -->
 
-![stratum logo](stratum/docs/images/stratum-logo.png)
+<p align="center">
+  <img alt="Stratum Logo" src="stratum/docs/images/stratum-logo.png">
```

Comment:
Surprisingly, this works both on GH and with VSCode preview.

Reply:
They both from Microsoft 😏